### DISCUSSION TITLE
Split the training set and the test set

### DISCUSSION THREAD

**Discussion by imauzzc** [pycaret/pycaret#2985] [Q&A]
Hello, is it possible to manually split the training set and the test set?

My experiment was repeated three times and I want to use two of them as training set and the other repetition as test set, is this OK? If it is possible, please tell me the method, thanks a lot!
